Database/Models/BanRecords.cs

```diff
@@ -0,0 +1,31 @@
+using System.ComponentModel.DataAnnotations;
+
+public class BanRecords
+{
+    [Key]
+    public int BanId { get; set; }
+
+    public ulong GuildId { get; set; }
+
+    public ulong UserId { get; set; }
+
+    public ulong ModeratorId { get; set; }
+
+    [MaxLength(500)]
+    public string Reason { get; set; } = "None";
+
+    public string ReferenceImagePath { get; set; } = "None";
+
+    public ulong? ReferenceMessageId { get; set; }
+
+    public DateTime BanDate { get; set; } = DateTime.UtcNow;
+
+    public DateTime? ExpirationDate { get; set; }
+    [MaxLength(50)]
+    public string AppealStatus { get; set; } = "None";
```

Comment:
What is this variable? It doesn't make much sense.

If you want to store **whether or not the user wants to appeal**, use a boolean.

If you want to store the **appeal message** (and with that, whether or not the user wants to appeal), call it `AppealMessage`, increase the max length to something more reasonable for a ban appeal (500 or more), and make it nullable (if there is no appeal, it should be `null` and not `"None"`)

Though it's also worth noting that this appeal system should later be integrated into our custom ticket system, once we have one. And once we have that, maybe just storing whether or not the user has appealed with a boolean is a better choice. I recommend doing that.